frontend/src/service/Testimonial.jsx

```diff
@@ -13,7 +13,20 @@ async function getTestimonials(setTestimonials, setToast) {
 
     if (response.ok) {
       const data = await response.json();
-      setTestimonials(data);
+      let _data = [...data]
+      await data?.map((item,index) => {
+        let formattedPath = item.image?.replace(/\\/g, "/");
+        if (formattedPath?.startsWith("uploads/")) {
+          formattedPath = formattedPath.replace("uploads/", "");
+          if (formattedPath) {
+            formattedPath = `${END_POINT}/${formattedPath}`;
+          }
+        }else{
+          formattedPath = "./images/testimonialImg.png";
+        }
+        _data[index].image = formattedPath;
```

Comment:
@Kajol-Kumari  This is the function we used to get data. here I made small chage if image exist. we use that url else we use default image a url.